xsuite-chainsimulator/build-chainsimulator.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+VERSION="v1.7.5"
+
+if [ "$PLATFORM" = "darwin" ]; then
```

Comment:
The script could take 2 arguments: (1) the name of the final binary, (2) the ldflags. This way, no need to have a if/else in this script.

Reply:
Not done

Reply:
The ldflags are the same for both Linux and Mac hence no need for another variable. If I try to use the Linux flags that work in the simulnet, this will not compile on Linux, so the same flags are used for both.

Reply:
Done